src/styles/main.scss

```
@@ -1,3 +1,73 @@
-body {
+$calendar-border: 20px 10px;
+$calendar-gaps: 1px;
+$day-size: 100px;
+$day-background: #eee;
+$day-background-hover: pink;
+$day-border-color: #000;
+$day-font: arial, helvetica, sans-serif;
+
+$day-of-week:
+  "mon" 0,
+  "tue" 1,
+  "wed" 2,
+  "thu" 3,
+  "fri" 4,
+  "sat" 5,
+  "sun" 6;
+
+body,
+p {
   margin: 0;
 }
+
+.calendar {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  gap: #{$calendar-gaps};
+  padding: #{$calendar-border};
```

```suggestion
  gap: $calendar-gaps;
  padding: $calendar-border;
```
You can use variables like this